CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ the detailed section referring to by linking pull requests or issues.
 
 #### Minor
```

Comment:
![image](https://github.com/sovity/edc-ui/assets/30114507/fda2ff13-d9a9-43d9-ae1c-d58617193e1c)

should be called "Sovereign Legal Name" or have a description that describes that it is the Legal Name of the Data Sovereign